tests/test_assessment_engine.py

```diff
@@ -1,5 +1,6 @@
 """Unit test module for Assessment Engine API"""
 import json
+import sys
```

Comment:
I think this is leftover from when you were checking python versions, doesn't look like you need it anymore

Reply:
Good catch